gtfs_rt_validator_api.py

```diff
@@ -239,12 +239,12 @@ def validate_gcs_bucket_many(
     project_id: str = "cal-itp-data-infra",
     token: str = None,  # "cloud",
     param_csv: str = f"gs://gtfs-data-test/rt-processed/calitp_validation_params/{pendulum.today().to_date_string()}.csv",
-    results_bucket: str = "gs://calitp-py-ci/gtfs-rt-validator-api/test-pipeline",
+    results_bucket: str = f"gs://gtfs-data-test/rt-processed/validation/{pendulum.today().to_date_string()}",
     verbose: bool = True,
     aggregate_counts: bool = True,
-    status_result_path: str = "gs://calitp-py-ci/gtfs-rt-validator-api/test-pipeline/status.json",
+    status_result_path: str = f"gs://gtfs-data-test/rt-processed/validation/{pendulum.today().to_date_string()}/status.json",
```

Comment:
Same as above, do we need `get_bucket` here?

Reply:
yes!